ProjectGenerator/bin/ihp-new

```diff
@@ -83,7 +83,9 @@ if ! [ -x "$(command -v direnv)" ]; then
 	read -r -n1 choice
 	echo ""
 	case "$choice" in
-	y | Y) nix-env -iA ${NIXPKGS_ATTR}.direnv ;;
+    y | Y)
+        nix profile install nixpkgs#nix-direnv
```

Comment:
If I'm not mistaken, [`nix profile`](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-profile) needs the experimental features `nix-command` and `flakes`activated, so this would fail on a system without them activated beforehand. We should probably add `--extra-experimental-features` similarly to the other nix commands below in this file.

Reply:
This has been fixed later on master after this PR

Reply:
Ah oh, I see, thanks